Documentation/ceph-object.md

```diff
@@ -224,19 +225,30 @@ To test the `CephObjectStore` we will install the `s3cmd` tool into the toolbox
 yum --assumeyes install s3cmd
 ```
 
+Configure s3cmd `s3cmd --configure` with the ACCESS_KEY and other details we got as described here - https://github.com/s3tools/s3cmd
+
+
+### List the generated bucket name (Note that to keep the bucket name unique we created the bucket with `generateBucketName`.
+Please see here for more details regarding OBC creation [Object Bucket Claim](ceph-object-bucket-claim.html)
```

Comment:
Since the OBC is discussed earlier in this doc, I don't think we need another link here. How about removing this line? All we're doing is listing buckets in this section, so OBC isn't really necessary, right?

Reply:
the OBC document is very detailed and I thought providing the link here may help people to check the different possibilities.

Reply:
I was referring to the "Create a bucket" section above in this same document that already mentions OBCs. But the point of this section is just to list buckets. To keep it simple, it seems sufficient something like this:
```
### List buckets

To see what buckets have been created, list them with this command:
```